magefile.go

```diff
@@ -0,0 +1,71 @@
+// Build a script to format and run tests of a Terraform module project
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+)
+
+// The default target when the command executes `mage` in Cloud Shell
+var Default = Full
+
+func main() {
+	Default()
+}
+
+// A build step that runs Clean, Format, Unit and Integration in sequence
+func Full() {
```

Comment:
nit: more clear if named something like `FullTestSuite` or `RunAllTests`

Similar -- `Unit` and `Integration` may be more clear if its `RunUnitTests` or `RunIntegrationTests`

... total nit. ignore if you want.